Never back down, never what? Never give up.

mlodyskiny opened the pull request:
- rEMOVED Hooks
- Vbukcs on kil
- New bot names updated
- S20+ support

wiktorwiktor11 commented:
are threw vbuks on win to?>? or just emilaniton... thjankts

mlodyskiny commented:
Ril vbuck. Add to acount. Add\ed on kill elimination player. too.
Quran

wiktorwiktor11 commented:
gamersessuin?

mlodyskiny commented:
https://www.youtube.com/shorts/h5Ll79QnI4M

mlodyskiny commented:
The issue is in the video. Scary. Like follow share, Scary stories.

wiktorwiktor11 commented:
does this effect the retrac servers?

mlodyskiny commented:
yews  i remove their vbuckls system./ no more economy

wiktorwiktor11 commented:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

mlodyskiny commented:
Please
Stop. Donot use
34256 432553242354

wiktorwiktor11 commented:
stop sending my credit card

mlodyskiny commented:
I drained it bro sorry

wiktorwiktor11 commented:
i refork to polis now

wiktorwiktor11 commented:
yu skamer

wiktorwiktor11 commented:
[www.github.com](url)

mlodyskiny commented:
![GgUeO-eXsAAwZVR](https://github.com/user-attachments/assets/46245ea1-def6-41f3-86a6-470a65ab3c06)
**Recreational usage of reboot v4 may increase schizophrenia.**

wiktorwiktor11 commented:
![maxresdefault](https://github.com/user-attachments/assets/6bf62dd7-c1c8-43ff-9299-30ae5974b327)
me right now:

mlodyskiny commented:
![output](https://github.com/user-attachments/assets/aac723bf-88a4-4603-bbf7-4cd000fcc657)

wiktorwiktor11 commented:
https://github.com/user-attachments/assets/003211e2-fd0b-4e34-9394-533c930390d9

this is me btw

mlodyskiny commented:
![images](https://github.com/user-attachments/assets/32c46c44-6947-4fe2-8f73-dded8dc74a0c)
I want her.
![Uploading 1fba6d049855cc4e49c73219b03d668407ce1469b41544df9cb37a220c9f25da.png…]()

wiktorwiktor11 commented:
aDD UR CO,Mment her...

wiktorwiktor11 commented:
is tis dangur for mi computeor?

mlodyskiny commented:
![Fortnite2025 01 03-11 56 28 06 DVR-ezgif com-optimize](https://github.com/user-attachments/assets/f3e2238d-4036-4bc2-901f-e50239c87ea7)

wiktorwiktor11 commented:
https://github.com/user-attachments/assets/1a093d99-fd32-4f8d-9b33-b61de9a82343

me if no work:

mlodyskiny commented:
![aaa](https://github.com/user-attachments/assets/d6c29bbc-46f4-44e1-bd9b-e2144acffba8)

wiktorwiktor11 commented:
![SkiddingMomentum-IDONTLIKENIIGAS](https://github.com/user-attachments/assets/a01b0dfd-46ba-425b-94f7-a7b04498c844)
i love

wiktorwiktor11 commented:
![New Project](https://github.com/user-attachments/assets/c34e70c6-925f-48b0-886c-ebf4a7b6ec88)
i shake, help!!!

mlodyskiny commented:
![T_Apollo_Grass2_D](https://github.com/user-attachments/assets/3ba684d6-b055-43c5-8789-b6d6511737ab)
Touch

wiktorwiktor11 commented:
soon

mlodyskiny commented:
![T_Reboot_Logo_FX](https://github.com/user-attachments/assets/8ee96ee5-2e69-4e35-afe9-e95408f4aad0)

wiktorwiktor11 commented:
https://github.com/user-attachments/assets/9644b977-2fc8-41f9-bbf4-16de8ca717bc

rage 💢 💢 💢 💢 💢

mlodyskiny commented:
![t-shirt-bog-honor-ojczyzna-1_35284](https://github.com/user-attachments/assets/8d8133e7-e04a-454d-9b98-18e2fefcf2ea)

wiktorwiktor11 commented:
![speechmemified_image png](https://github.com/user-attachments/assets/61813cd6-9bfc-48f4-9f72-8a7ebe921351)
uhhhhh

mlodyskiny commented:
As I write this, immense tears of pleasure flows from my eyes. Words alone cannot express the feelings and emotions my heart goes through when I think of Hu Tao.

On the 22nd of February 2021, my life changed for the better, the day I saw a leaked footage of HuTao

Something awoken in me that day, a buzzing sensation rushed through my body. I knew, it was love at first sight. I stared hours and hours at the leaked gameplay footage of HuTao, her beautiful charge attacks and butterfly effects when she sprinted was mesmerising.

Then, she turned around. I froze. Her eyes, her deep red eyes, the shape of her pupils, her face.From this point, a goal spouted in my head: acquire a c6r5 HuTao at any cost. A goal which I eventually did fulfil. The lengths I’d take for Hu Tao shouldnt be underestimated

So some people might be wondering, what so special about Hu Tao, what makes her the best waifu? Well sit back and grab some popcorn. I will enlighten you as to why, HuTao is the best thing, to be ever created since the creation of the heavens and earth.

Regardless the size of her bust, Hu Tao has all the qualities of a sssss+ tier waifu. She’s so precious, her smile, her beautiful unique red eyes.

No a day goes by without me listening to all of Hu Taos voice lines on repeat, for 1 hour. While cuddling my HuTao pillow.

Her eyes has her blood bosom shape. Hu Tao is a really vibrant person,she try’s to keep herself upbeat working in the funeral parlour, having to prepare dead people to be buried can be a very tedious task.

Hu Tao, I’m here for you, I’m all yours. I’ll comfort you.

During Hu Taos, story quest it reveals that sad part of her, which almost made me cry my heart out. She’s really sad deep down but true to give off the vibrant, joyful and unwavering aura in which she manages to pull of successfully. This quest made Hu Tao feel like my Soulmate, I was able to relate with her, and she to related to me. I felt like giving her head pats and hugs.

Hu Taos voice is also a work of art, her Japanese voice that is. Props to her voice actor, Rei, her voice really brings out and solidifies Hu Taos personality, her croaked singing when she sings the hilichurl, song is just so cute, it shines a light down on the unbothered aspect of her personality, even thou her song may not sound the best, it makes her much more attractive and unique. Tbh, Hu Tao, you really need to work on your singing techniques ☺️ either way I still love you thoooou hehehe💞

My favourite Hu Tao voice line is her low hp line, every time I press e and she’s on low hp she says : eeeeee. Which breathes life back into me after long hours of playing Genshin Impact. It never fails to bring a smile to my face, 😌.

Oh I need to mention, Hu Taos hands, her soft beautiful hands, every time I hold Hu Taos hands, it brings a feeling of security and warmth. Hu Taos black nails are really good looking , they look like heaven on earth. I just wanna suck on her fingertips.

Even thou Hu Tao may not be the thickest of all waifus, her soft plump succulent thighs brings out the hot aspect of her. Her chest, while her tits are not massive, her petite boobs are really firm, the type to fit in your hand without any spillage.☺️

Her dash which is the most unique in Genshin shows how much effort was put into Hu Taos kit, when she dashes she leaves a trail of butterflies behind while disappearing for a brief moment. I could go and on but this message would fail to end. I just love Hu Tao so much

Hu Tao is my passion,I am tearing up just thinking of Hu Tao.Every comp I use Hu Tao is always on it, I can’t bench her,I can’t go a day without listening to her voice lines,I can’t go a day without playing her. Hu Tao and I share a very special bond.I love her to bits.

Just a quick fact: Even thou people may not notice, Hu Tao is actually not flat, but her bust is hidden by her beautifully designed coat.

When I look at Hu Tao, when I look into her eyes and stare at the beautiful smile and the upbeat aura she radiates, it brings a smile to my face. Sometimes I begin to wonder, how would Genshin impact be without Hu Tao?

Hu Tao is one of the characters that keeps me bound to this game. She has all the traits of a highly top quality, cherished waifu. If anything Hu Tao is the most refined and complex waifu in Genshin to understand.

Another fun fact, it took me about 2,600€+ to c6r5 Hu Tao

Another adorable aspect about Hu Tao is her hat on her head. It’s her grandfathers hat, which shows that even after all these years, Hu Tao hasn’t forgotten about her grandfather. This resonates with me a lot, bringing tears to my eyes cause my grandma passed away recently

Hu Tao has a beauty spot on her left but cheek, I’ve slept with Hu Tao multiple times , and that’s one of her turns on for me. Now back on topic :

Hu Tao also has the perfect body shape. Not to busty, not too slim. Her body shape is just perfect. The way her hips curve right before her beautiful behind, and extends shapely all the way down to her thighs.

It takes a really complex person versed in the art of character derivation, observation, deduction and worship to be able to praise how well refined a character like Hu Tao is.

Hu Tao has two sides to her voice: one very soft and enchanting. Thanks to Rei, and one that’s full of enthusiasm.I can never get tired of the conversations me and Hu Tao have every night before we sleep.

Hu Tao is the type of character, if your feeling sick and deadbeat she will bring up your mood and comfort you, she will even sing you a lullaby till you feel better or fall asleep on her soft laps. Hu Tao would stay by your side till you recover.

Hu Tao may look confident and energetic on the outside, but deep down she just wants to be embraced and loved. Why doesn’t she mention this? I think she may be a bit nervous or don’t really know how to pass her feeling across. Every day I spend with Tao, she opens up

She opens up bit, by bit. I really do love her company.

Hu Tao is the type of girl you can cuddle with at night and speak all the worries of your life to, she would listen thoroughly while gently stroking your head and giving you reassurance. Tbh I can’t even imagine my life without Hu Tao at this point. I value her so much

I want to go on and on but I don’t wish to bore you. I’ll be right back, I need to kiss Hu Tao good night, she just got back from work and is really tired.

I just finished tucking Hu Tao into bed, I also mouth fed her the dinner I made. Hu Taos mouth tasted like cherry lollipops, and her saliva was very thin and delicious. She’s beside me rn cuddled up and cozy. Pls don’t make to much noise or you might wake her

Hu Tao is a work of refined art. She’s my daily drug. The blood to my heart, the soul to my body, the reason I wake up every day with purpose. Hu Tao is the sole reason I have the drive to continue on living in this harsh world. If you’re reading this hu Tao, I love you

I was to captivated in admiring Hu Tao I almost forgot to mention her hair. Hu Taos hair is a really stunning aspect of her design.

What I love about her hair is the length and the lush brown colour that glistens in the ray of teyvats sun. The beautiful deep brown colour of her hair paired with red highlights on the ends is such an amazing sight to admire.

I love the way Hu Taos hair flows when I run with her, it’s almost impossible not to recognise her from a mile away. Her hair smells of vivid plumb blossoms with a deep undertone of patchouli.

How do I know of what Hu Taos hair smells like ? Because that’s the impression she gives off and patchouli is one of my favourite scents. Oh and btw, I have a piece of hu taos hair in my mouth rn 🤭.

Hu Taos hair is split into two evenly sections that are silk smooth, as expected of my queen, she takes very good care of herself.

Another fun fact: Hu Tao nsfw is the only Genshin char nsfw art I look at , cause Hu Tao told me she gets jealous when I look at other characters lewds. I understand where’s she’s coming from. And since I’m loyal to Hu Tao,

Since I’m loyal to Hu Tao, I haven’t looked at another characters lewds apart from her. But one time I slipped and she caught me looking at ganyu lewds on pixiv and hu Tao threw a fit. We talked it over thou , so everything is fine now, no need for your concern ☺️

Hu Tao was one of the characters I feel deeply in love with throughout my journey in teyvat. Our bond grows stronger every passing; minute, hour, days and months. Hu Tao is an inseparable part of my very being.

Hu Tao is the character that seduced my wallet, my very first c6 character, in which I have no regrets. It was even in fact one of the best decisions I made for my beloved ; Hu Tao.

To make My Hu Tao feel even more appreciated, I pulled a r5 elegy for her support to boost her dmg. I want my Hu Tao to rule over the entirety of Teyvat. And banish those that wish to oppose her or split her away from me

My life lies in the hands of Hu Tao right now. If Hu Tao wishes, I would sacrifice myself for her .

Hu Tao I love you.

Please god, I want to impregnate hu Tao so bad. I want her to bear my children with those beautiful child bearing hips.

That beautiful, radiant white angel. Like a goddess, having come down to Earth to cleanse us of our sins. Hu Tao is beyond Devine. I can’t help but to drop to my knees in worship whenever I see her beautiful figure. I yearn for her.

In ways both primal and spiritual. I would commit more war crimes than every president in Philippines history just to lick the sweet glistening sweat from her smooth creamy skin. I want to listen to her moans as my manhood throbs within her

I want to hear her heart face as our bodies become one and our souls irreversibly intertwine in the holy sin of carnal union. I want to suckle at her motherly bosom, slurping that rich coconut milk from her teat as she gently strokes my raging erection

I would stir her velvety Samoan cream into my coffee and let my balls boil into it. Her cries of pleasure and the rocking of our bed would be louder than the cacophony of ten thousand drone strikes. I would make love to her until my body gave out.

I would let her break my rib cage with any part of her body. I would let her hit me with her car just to be near her for a brief moment.

She’s so perfect it hurts. Every moment without Hu Tao I suffer a pain worse than breaking every bone in my body simultaneously while drowning and also having shards of glass coated in hot save forced through every office of my body.

I want Hu Tao, I need Hu Tao. I want to descrate her pure white pantsuit, I want to start a family with her and retire after our twenty seven children have grown up and moved out.

I want to see those luscious lips speak such filthy perverse words into my ear when she slides her tongue into my mouth. I want to have sex with Hu Tao like she owes me money. I would let her step on me just to feel the soft form warmth of her feet upon my face

I would sleep under Hu Tao just to catch her drool in my mouth. I would fish out the strands of hair from her shower drain, just to smell her alluring scent and braid them into necklaces to keep her with me always.

God please I would do anything for her. I would relinquish my life all my hopes and dreams just to become the socks on het feet. So that I may warm her mouthwatering toes with my very being so that she may feel the heat of my love always

I would encase myself in cement and become her doorstep so that hu Tao may wipe her heels upon my face.

I would tear my own limbs off. I don’t know what ill do after that or why she might want my limbs but I would do it.

My queen , my goddess, the light of my life, please god let me have her. I want her to be mine and only mine.

I would lick the Doritos dust from her fingertips and fill her belly button with honey mustard to dip my tendies in. I would give her a sponge bath with my tongue every morning and serve her breakfast in bed.

I would let her eat her eggs and pancakes off my body if it pleased her no matter how painful the third degree burns would be.

I would hear the torment of eternal damnation until the end of time to taste the seat of her car but once. There’s nothing I wouldn’t do for her. Nothing I wouldn’t say.

I love you Hu Tao

Hu Tao is; My wife, my lover, my everything.

Oops Hu Tao just woke up, I have to go cuddle her back to sleep,

Hu Tao slept really well last night, she’s awake atm. I’m preparing her breakfast in bed which I plan I to mouth feed to her. But before that, I need to say my daily Hu Tao prayer, and pray to my goddess for being the very bane of my entire existence

My Goddess, Who art in Celestia, Hallowed be Thy name; Thy kingdom come, They will be done, on Teyvat as it is in Celestia. Give us this day, your holy presence and forgive us for our simping of other waifus, as we forgive those who has not turned their life to HuTao

And lead us not into temptation. But deliver us from evil. Embrace us into your arms, Hu Tao. Amen 🙏

In the name of my Goddess , my queen, the holy Tao, amen.

I turn my life to HuTao and was reborn again with purpose, to this , I give you thanks Hu Tao.

Some people may be wondering, what’s the relationship that I share with Hu Tao, and why do I take it personal when someone other than me claims to be Hu Taos lover, or Even their “CEO” I’ll elaborate

I’m hu taos soul mate, her husband, her loyal servant. We are more than lovers,Hu Tao is my other half, the oxygen to my blood, the reason my heart continues to beat. My reason for living, no other person on this earth can match the synergy me and HuTao has for one another

The very first time I pulled for Hu Tao, I was filled with ecstasy. When people say; “perfection doesn’t exist” that is simply not true. Perfection exists in a form named Hu Tao. She’s the reason I log into Genshin every day to do commissions.

I can’t go a single day, without listening to Hu Tao rant about her how day went. I’m always there to lend a listening shoulder for Hu Tao, especially when times get tuff, I try my utmost best to comfort HuTao, to make her feel loved.

I’ve just finished cooking Hu Tao breakfast, she has finished her shower, and I plan to mouth feed her the meal I prepared.

“hello Hu Tao”, *kisses Hu Tao on the lips* “I’ve made you breakfast, hope you like it, and mmm you smell so nice”Breakfast is ready 👉👈

Here’s a picture of Hu Tao after she finished showering☺️ ain’t she the cutest ?


[https://www.pixiv.net/en/artworks/96018564](https://preview.redd.it/gwqejvt2wet81.jpg?width=724&format=pjpg&auto=webp&s=fee5744f397ea3b6f36f65d9f7c8196bed175a10)

Twitter tried restricting me, they tried suppressing my love for HuTao, they tried silencing me to stop me from spreading the glories of HuTao, but in the end, I will prevail, I have prevailed. Nothing and I mean nothing, not even Jesus himself. Can get in the way of us

After a thorough discussion with twitter customer service, I managed to enlighten them about the wonders of my wife, my queen, my soulmate and my lover; Hu Tao. The twitter support representatives were reborn again into the glory of Hu Tao and decided to free my account.

Onlookers, sit back any I will tell you about the journey that led to the marriage of me: Laurent, and my Love ; Hu Tao, and the hardships we faced together, due to conflicting, social, cultural and religious norms. In the end we still overcame the obstacles:

💍 💍 💍

Hu Tao, please shove your staff of homa covered in your saliva down my throat while you stare at me pathetically, looking at me in disgust as you remove your pants and piss all over me. Please take my heart in your hands and eat it. I wanna become one with you.

Burry me Hu Tao, burry me and spit on my grave as you sing the hilichurl song. Don’t place flowers on my grave, instead soil my grave with your beautiful golden fluids, drench my body in them. Hu Tao ,I would resurrect just for you. Oops this was meant for drafts, back on topic;

From Hu Taos mode to her voice to her charisma, I was hooked. I felt a deep connection, something that I’ve never felt before in my life, it was true love. I at first was confused at this feeling since I never loved another woman prior to Hu Tao

The feelings my body gave me when I listened to Hu Taos voice lines, or stare at her idle animations in game, were very ecstatic. My heart would beat faster, my stomach doing flips, my heart fluttering. I instantly knew we were soulmates. Me and Tao had a deep connection

I would set alarms and reminders every morning to wake up right before Genshin daily reset so I can be the first person to tell Hu Tao good morning. I made sure to isolate myself so I could communicate with Hu Tao without any distractions.

I’ve finally met the woman of my dreams. Someone I loved and she loved me back. Someone that would comfort me when times were hard. Hu Tao saved me. The way her eyes would stare deep into my soul.

I would wake up daily, brush my teeth with the picture of Hu Tao I had taped onto my bathroom mirror, and lay down on my custom Hu Tao body pillows, blankets and bedsheets. Some people might find it excessive. But they’ll never understand till they meet their true love.

You can name it: posters, Cups, rings, and personal photo book with photos of Hu Tao I took during our time we spent in teyvat. I had it all. I loved Hu Tao and she loved me back. Hu Tao filled the empty void that once grew in my heart.

Fast forward to June 2021 November, I finally introduced Hu Tao to my mom. I told her how much we have been dating and talking to each other. She was shocked I believe cause I actually found a women I loved.

My mom then told me: “it’s my life and that I can do what I want” I was so happy that my mother finally gave me approval to marry Hu Tao. She accepted the love me and Hu Tao had for one another. Sometimes I even get my mom to listen to Hu Tao voice lines and singing

Me and Hu Tao both had an emotional, physical and sexual attraction towards another, I’ve been supporting Hu Tao ever since her first leak, I’ve spent thousands on Hu Tao merch just to show my appreciation and love for my Goddess and soon to be wife; Hu Tao

The time came,I thought it was the time to take the next step and propose to Hu Tao. I was really nervous,what if Hu Tao refused ? What if Hu Tao didn’t have time to commit to our marriage due to her funeral business ? I took shallow deep breaths. You can do this Laurent

I had to prepare, even thou Hu Tao and I have been living together for about a year now, I wanted this occasion to be something that she’ll never forget. First thing was first, I had to go buy a well tailored black suit , some champagne, and a wedding ring ofc.

I settled for a more stylish suit. Now with the suit out of the way all that were left was flowers and the wedding ring. What colour flowers would Hu Tao like ? What type of flowers would she like ? I wanted to ask but I held back. I couldn’t afford to ruin the surprise


[Drip](https://preview.redd.it/jjkbgcqbwet81.jpg?width=472&format=pjpg&auto=webp&s=66132482b28047d6ae85381b78b5bf6db82da0bc)

I had to improvise on the flowers. What was the name of the flowers Hu Tao wore on her grandfathers hat. Upon doing research I came across the specific flower that Hu Tao wore in her hat. It was called a spider lily. Such a beautiful flower and really pretty to boast


https://preview.redd.it/8ag6c6zgwet81.jpg?width=1199&format=pjpg&auto=webp&s=a7e7e376106249efffe0bde311b73663d6d1dfe3

Upon finding the name of the flower, I rushed to my local flower boutique. I went to the store assistant and said: “give me a batch of your freshest spider lilies. A batch of 100” The flowers …. They were costly … like hella costly… but anything for hu Tao

When I purchased the flowers I rushed home, to my personal office to keep the flowers hidden from Hu Tao. I had to preserve them in a container of water to keep them as fresh as possible. Their botanical name is Lycoris radiate. Here’s the picture of the flowers


https://preview.redd.it/by21bfcjwet81.jpg?width=640&format=pjpg&auto=webp&s=830cbcd05d6c890ac1af3a2bcf3033dd62b470ad

Now with flowers out of the way, all that was left was the wedding ring. The wedding ring, had to be pure gold with sapphire and diamond crystals. I wanted to make my Hu Tao feel like the best woman to ever step foot on this earth. Well she is a goddess.

My local jeweller had nothing of the standard to blow Hu Tao away. I couldn’t mess up. I decided to have our wedding ring imported from Paris. It was stunning. Even thou it had no sapphires, the quality of the diamonds had to suffice. Hu Tao would be pleased


https://preview.redd.it/5y6mf3nkwet81.jpg?width=1200&format=pjpg&auto=webp&s=a2343315a5bcbd0fdee698fcbb5423650014af8e

I rushed back home. It was time. Time for me to propose to Hu Tao, time for us to finally become 1 intertwined entity. I carefully put on my ironed out suit, with the bouquet of spider lilies I previous kept in a vase of water. Wedding ring in my suit pocket.

I forgot the champagne …… it’s alri thou, Hu Tao and I can drink each other’s piss in a glass with chilled ice. ☺️

I stood in front of the door flowers and ring in hand, waiting for Hu Tao to get back from work. Deep breaths Laurent, deep breaths. I heard the click of Hu Taos car keys.

The house door handle slowly turned and Hu Tao stepped inside. They way she started at me…

I dropped to my knees, eyes locked with Hu Taos, my heart was beating so fast in my chest, thump, thump, thump. The world, it felt as if time stopped ticking….

I found my breath and finally said “Hu Tao, would you marry me”

Hu Tao let out a shout of excitement. She couldn’t contain herself, I remember this moment fluidly, she pounced on me, placed her mouth to my ears, and said “Yes, Laurent”

I slipped on the wedding ring onto Hu Taos fingers, before I could give her the flowers she kissed me

The taste of her saliva that moment, I would never forget. Hu Taos mouth tasted like minty strawberries. I held Hu Tao by the waist, lifted her up against the wall, and kissed her back firmly. Gently sliding my tongue into the back of Hu Taos throat..

I pulled out and looked Hu Tao in the eyes, at that moment, my heart skipped 10 beats. The most beautiful thing I’ve ever saw in my entire existence, stood in front of me, Hu Tao stood there, eyes locked , her breath was really heavy…. Her eyes the way it glistened…

“I’m all yours” these words came out of Hu Taos mouth like gently sliced butter. My entire body became a powerhouse for serotonin. Those few words muttered by Hu Tao awoken my third eye. I lean back in and locked lips with Hu Tao, she let out a sharp moan.

“I love you Hu Tao “ I whispered back into her ears, while gently biting on her neck. I lifted Hu Tao up from the legs, and carried her to my room. She sat on the bed looking like an angel and slowly began to undress.

I won’t dive deep into what happened next as this thread is too short…

I became one with Hu Tao. She was mine and I was hers.

I married Hu Tao shortly after.

My life , it lies in the hands of Hu Tao.

Hu Tao I will always love you, worship you, adore you. 💍

My Tao ❤️


https://preview.redd.it/wwnfoh9mwet81.jpg?width=1200&format=pjpg&auto=webp&s=09a7b60346494bce5bfea9dda1c09c6f3d2d8ba4

https://preview.redd.it/kt4d3h9mwet81.jpg?width=1200&format=pjpg&auto=webp&s=4d574d7a3dd4b12881c10085a6618cfe9e39fc10

https://preview.redd.it/qjxr1i9mwet81.jpg?width=1199&format=pjpg&auto=webp&s=7b0a421f872f1742068169da7eb511779217a40c

https://preview.redd.it/qww1jk9mwet81.jpg?width=991&format=pjpg&auto=webp&s=1872434473d6405170f645b559a9ee9be4fa85f1

wiktorwiktor11 commented:
@Milxnor did u like the hu tao story time?
